sdk/core/azure-core/inc/azure/core/internal/http/pipeline.hpp

```diff
@@ -55,6 +57,100 @@ namespace Azure { namespace Core { namespace Internal { namespace Http {
       }
     }
 
+    /**
+     * @brief Construct a new Http Pipeline object from clientOptions.
+     *
+     * @remark The client options includes per retry and per call policies which are merged with the
+     * service-specific per retry policies.
+     *
+     * @param clientOptions The SDK client options.
+     * @param telemetryServiceName The name of the service for sending telemetry.
+     * @param telemetryServiceVersion The version of the service for sending telemetry.
+     * @param perRetryPolicies The service specific per retry policies.
+     */
+    explicit HttpPipeline(
+        ClientOptions const& clientOptions,
+        std::string const& telemetryServiceName,
+        std::string const& telemetryServiceVersion,
+        std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> const& perRetryPolicies)
+        : HttpPipeline(
+            clientOptions,
+            telemetryServiceName,
+            telemetryServiceVersion,
+            std::move(perRetryPolicies),
```

Comment:
what's the point of `std::mov`ing from a const?

Reply:
My bad, I will make the arg &&.  
The servicies will create and allocate the vector of policies and we just want to move the vector